pkg/domain/entities/images.go

```diff
@@ -318,3 +318,31 @@ type SignOptions struct {
 
 // SignReport describes the result of signing
 type SignReport struct{}
+
+//ImageMountOptions describes the input values for mounting containers
```

Comment:
nit: `s,//Image,// Image,` (i.e. missing space after the `//`)

Reply:
Fixed